jekyll/_cci2/env-vars.md

```diff
@@ -49,6 +49,47 @@ Running scripts within configuration
 may expose secret environment variables.
 See the [Using Shell Scripts]({{ site.baseurl }}/2.0/using-shell-scripts/#shell-script-best-practices) document for best practices for secure scripts.
 
+
+### What Environment Variables Look Like In An Example Config
+
+Consider the example `config.yml` below:
+
+```yaml
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.0-browsers
+    steps:
+      - checkout
+      - run:
+          name: "Setup custom environment variables"
+          command: |
+            echo 'export MY_ENV_VAR="FOO"' >> $BASH_ENV
+      - run:
+          name: "What branch am I on?"
+          command: echo ${CIRCLE_BRANCH}
+      - run:
+          name: "What branch am I on now?"
+          command: echo $CIRCLE_BRANCH # Note: you can interpolate vars with or without brackets
+      - run:
+          name: "What was my custom environment variable?"
+          command: echo ${MY_ENV_VAR}
+```
+
+In the `config.yml` above we are: setting our own custom variables, using
```

Comment:
remove 'we' and 'our' these pronouns could be confusing for readers, add 'a' before built-in, change to demonstrates-> The config.yml above sets a custom variable using a built-in...

Reply:
I updated this to turn it into a bulleted list - let me know if it's alright!